server/helpers/helperFuncs.js

```diff
@@ -0,0 +1,105 @@
+const db = require('./../db');
```

Comment:
Coool! You realized you needed a place to put all code you need in several places or that didn't belong to components, this is good. 

Seeing the meaning of the different functions, I think that instead of a helper, this is more like a mix of `Authentication` and `Game` services.